### ISSUE TITLE
Available expression on zip_selected not set

### ISSUE THREAD

**Issue by shylux** [4teamwork/ftw.zipexport#25]
Should be hidden if zipping is not allowed on this type.
The same as: https://github.com/4teamwork/ftw.zipexport/blob/master/ftw/zipexport/profiles/default/actions.xml#L12

**Comment by jone** [comment]
No: the container is not relevant at all, only the things I select to export must be zipable.

But the [available check](https://github.com/4teamwork/ftw.zipexport/blob/master/ftw/zipexport/browser/zipexportview.py#L48-L49) is wrong, it should check the selected objects for the `ZipSelectedExportView` and the current context (current implementation) for the `ZipExportView`.

@shylux can you fix that?
